docs/source/conf.py

```diff
@@ -0,0 +1,81 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath("."))
+
+import neuralcompression
+import pytorch_sphinx_theme
+
+# -- Project information -----------------------------------------------------
+
+project = "neuralcompression"
+copyright = "2021, Facebook"
+author = "Facebook"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.0"
```

Comment:
Could we import the version from `neuralcompression`?

Reply:
Done!